### PR TITLE
Global resource

### PR DESCRIPTION
**Pull request by bergman**

### CODE REVIEW

File: styx-scheduler-service/src/test/java/com/spotify/styx/SchedulerTest.java [hunk 1]
```diff
@@ -337,6 +337,22 @@ public void shouldFailWhenUnknownResourceReference() throws Exception {
   }
 
   @Test
+  public void shouldIssueInfoIfGlobalResourceDepleted() throws Exception {
```
Comment (TinaRanic) [inline styx-scheduler-service/src/test/java/com/spotify/styx/SchedulerTest.java]: shall we also test the fallback, when the global resource doesn't exist in the storage?

Comment (bergman) [inline styx-scheduler-service/src/test/java/com/spotify/styx/SchedulerTest.java]: good suggestion, done and pushed :)

**Comment by TinaRanic** [comment]
👍 LGTM

**Comment by bergman** [comment]
I thought a bit and am not so sure anymore that the default case should be to limit if not set. I think I would rather keep the original behaviour of running without any resource if not set.

File: styx-scheduler-service/src/test/java/com/spotify/styx/SchedulerTest.java (hunk 1 above)
Comment (bergman) [inline styx-scheduler-service/src/test/java/com/spotify/styx/SchedulerTest.java]: went back on this, no fallback, so no test.

**Comment by fabriziodemaria** [comment]
Close this? @bergman

**Pull request closed without merging**